src/main/scala/info/bethard/timenorm/formal/Readers.scala

```diff
@@ -54,7 +54,8 @@ class AnaforaReader(val DCT: Interval)(implicit data: Data) {
       } else if (value.forall(_.isDigit)) {
         IntNumber(value.toInt)
       } else {
-        VagueNumber(value)
```

Comment:
I don't understand why this should be an error. Even if it's "2-3", isn't VagueNumber still an accurate description? If VagueNumber is giving you a problem in the scorer, maybe throw the exception in the scorer?